velocity-engine-core/src/main/java/org/apache/velocity/runtime/RuntimeInstance.java

```diff
@@ -384,6 +400,20 @@ private void initializeSelfProperties()
         hyphenAllowedInIdentifiers = getBoolean(PARSER_HYPHEN_ALLOWED, false);
     }
 
+    private char getConfiguredCharacter(String configKey, char defaultChar)
+    {
+        String configuredChar = getString(configKey);
+        if (configuredChar != null)
+        {
+            if (configuredChar.length() != 2 || configuredChar.getBytes(StandardCharsets.UTF_8).length != 1)
```

Comment:
I don't understand: `configuredChar.length() != 2`. What is the purpose here?

Reply:
Deprecated code from a previous attempt. Fixed.